### PR TITLE
Diversify sound effects for infiltrations on geoscape

### LINKED ISSUE #268: Use specifc sounds for new missions/CAs
Currently it uses the promotion sound which I put there just for the sake of testing. Needs to be replaced with sounds (like UFO, GOps, etc)

Blocked by #257 as it needs to be a field on the activity template

### PR DESCRIPTION
**Pull request by cannonfodder1**
Closes #268

### CODE REVIEW

File: CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration_UI.uc [hunk 1]
```diff
@@ -240,7 +240,7 @@ static protected function EventListenerReturn CovertActionCompleted(Object Event
 
 		HQPres = `HQPRES;
 		HQPres.NotifyBanner(default.strInfiltrationReady, MissionState.GetUIButtonIcon(), MissionState.GetMissionObjectiveText(), default.strCanWaitForBonusOrLaunch, eUIState_Good);
-		HQPres.PlayUISound(eSUISound_SoldierPromotion);
+		`XSTRATEGYSOUNDMGR.PlaySoundEvent(Activity.GetMyTemplate().CompleteSound);
```
Comment (Xymanek) [inline CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration_UI.uc]: This isn't the CA being completed per say, it's the UI notification for transition from CA to infil mission site.

So `CompleteSound` is at least badly named in this case

Comment (cannonfodder1) [reply CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration_UI.uc]: I'll change it to MissionReadySound

**Comment by Xymanek** [comment]
`MilestoneSound` is generic enough, so I'm mostly fine with letting it sit on the general activity template. But the `MissionReadySound` is definitely very specific to infils, so please move it there

**Comment by cannonfodder1** [comment]
Covert Actions have their own special completion sound, and for Assaults the available and completion sounds are the same. Therefore, I'm moving both MissionReadySound and CompleteSound to the infil specific activity template.

**Pull request merged**